test/cloud/yaml_test.rb

```diff
@@ -0,0 +1,58 @@
+require File.expand_path(File.join(__FILE__, '../..', 'test_helper'))
+require 'rubber/cloud/yaml'
+
+# NOTE: Must re-record one at a time for any test that invokes create_instance.
+# This is true until we allow povisioning arbitrarily, or if you expect the instances list to
+# be empty.
+class YAMLTest < Test::Unit::TestCase
+  context 'yaml' do
+    setup do
+      @dbfile = "/tmp/#{SecureRandom.uuid}"
+      # env = {
+      #   'database' => @dbfile,
+      # }
+
+      env = Rubber::Configuration::Environment::BoundEnv.new(env, nil, nil, nil)
+      @cloud = Rubber::Cloud::YAML.new(env, nil)
+    end
+
+    teardown do
+      FileUtils.rm(@dbfile)
+    end
+
+    should 'create instance' do
+      assert @cloud.create_instance('', '', '', '', '', '')
```

Comment:
I don't see what this is testing? It doesn't verify that anything was created, only that a truthy value was returned.